### DISCUSSION TITLE
Export/Import argument

### DISCUSSION THREAD

**Discussion by mbnoimi** [pocketbase/pocketbase#2122] [Q&A] [answered]
Is there any argument in CLI for import/export database structure/data instead of using web ui?

**Comment by hungcrush** [comment]
@mbnoimi no. only web UI ^^

**Comment by ganigeorgiev** [comment, marked as answer]
@mbnoimi If you want to import and export your collections configuration programattically you can use the [`migrate` command](https://pocketbase.io/docs/migrations/).

`migrate collections` will generate a migration file with full snapshot of your current collections configuration.

Migrations (both JS or Go) are applied automatically on `serve`.

---

Currently there is no built-in data import/export API or command (this eventually will be handled as part of #48). As a workaround for now you can use the [`sqlite3 .import` command](https://sqlite.org/cli.html#importing_files_as_csv_or_other_formats). You can also find an example in https://github.com/pocketbase/pocketbase/discussions/2105#discussioncomment-5386191.

**Reply by mbnoimi** [reply]
Thank you.
I can't wait to see implementing feature #48. It's exactly what I missing in PocketBase.
